### ISSUE TITLE
false positiv in k8s ingress secretname for CKV_SECRET_6

### ISSUE THREAD

**Issue by dominik-ba** [bridgecrewio/checkov#4815]
**Describe the issue**
False positive:
CKV_SECRET_6: "Base64 High Entropy String"
defining a a secret name in a Kubernetes ingress for TLS fails this test

**Examples**
Actual code snippet:

ingress.yml

```yml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: helloworld-ingress
# ...
  tls:
    - hosts:
        - hello.world.mycorp
      secretName: helloworld-cert
```

check result:
```
      File: /templates/ingress.yml:23-24
        Guide: https://docs.bridgecrew.io/docs/git_secrets_6

                23 |       secretName: hel************
```

Since this is only the NAME of the secret and not the value this should not cause a finding.

**Version (please complete the following information):**
2.3.141

**Comment by pndurette** [comment]
Can confirm this same issue in a non-Kubernetes manifest YAML file.
We reference a secret by its _path_ in our secret store and that path is flagged.
